ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/RemoteImage.kt

```diff
@@ -11,25 +11,31 @@ import coil.compose.AsyncImage
 import coil.compose.AsyncImagePainter
 import coil.disk.DiskCache
 import coil.request.ImageRequest
+import coil.transform.Transformation
 import com.revenuecat.purchases.ui.revenuecatui.UIConstant
 import com.revenuecat.purchases.ui.revenuecatui.helpers.Logger
 
+@SuppressWarnings("LongParameterList")
 @Composable
 internal fun RemoteImage(
     urlString: String,
     modifier: Modifier = Modifier,
     contentScale: ContentScale = ContentScale.Fit,
     contentDescription: String? = null,
+    transformation: Transformation? = null,
+    alpha: Float = 1f,
 ) {
-    AsyncImage(
+    return AsyncImage(
```

Comment:
Hmm I don't think we need the `return` here right?

Reply:
Nope, fixed